server/src/controllers/copilot.controller.ts

```diff
@@ -31,6 +31,6 @@ export class CopilotController {
   @UseGuards(JwtAuthGuard)
   @Post('api-key')
   async validateCopilotAPIKey(@User() user, @Body() body: { secretKey: string }) {
-    return await this.copilotService.validateCopilotAPIKey(user.id, body.secretKey);
+    return await this.copilotService.validateCopilotAPIKey(user.organizationId, body.secretKey);
```

Comment:
It is better to have current organization Id to be passed from frontend, as inferring organization id from the user can err when org switch isn't reflected incase of concurrent requests.